### PR TITLE
Docker: Full AF config_file support for baseline

### PR DESCRIPTION
**Pull request by psiinon**
Depends on https://github.com/zaproxy/zap-extensions/pull/3119

Also need to add integration tests - thats easier once the above PR has been merged.

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

**Comment by psiinon** [comment]
Changed to completely support the config_file option.
Depends on https://github.com/zaproxy/zap-extensions/pull/3150 - I need to sanity retest it after this has been merged and the live docker image rebuild.
Once this has merged I can test it with a config_url and hopefully then enable that option too in another PR.

**Comment by psiinon** [comment]
Have tested the included integration tests with the latest live image and they pass :D
So .. no longer WIP and tready to review!

File: docker/integration_tests/baseline_tests.sh [hunk 1]
```diff
@@ -62,13 +62,9 @@ echo
 echo "Baseline test 3 (vs example.com with INFO/WARN/FAIL and OUTOFSCOPE set)"
 /zap/zap-baseline.py -t https://www.example.com/ --auto -c configs/baseline3.conf > /zap/wrk/output/baseline3.out
 RET=$?
+# In this case use use the OUTOFSCOPE to ignore the rule that can get differing results ;)
```
Comment (kingthorin) [inline docker/integration_tests/baseline_tests.sh:65]: Double "use"

Comment (psiinon) [reply docker/integration_tests/baseline_tests.sh]: Fixed :)

**Review by kingthorin — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**